### ISSUE TITLE
Calculating wind u- and v-components from direction and speed

### ISSUE THREAD

**Issue by mpvginde** [harphub/meteogrid#19]
Hi,
I think there is a mistake in the calculation of the wind u- and v-components from wind speed and direction:  
https://github.com/harphub/meteogrid/blob/9da2345441037e82a518d4e7a18145285ae9a5e2/R/geowind.R#L49-L63

The `cos` and `sin`  directly use the  `direction` as angle. 
But this is the direction the wind is blowing from (meteorological convection).
The true angle to be used for the `cos` and `sin` function should be:
`270 - direction` (rotate wind 180 degrees + 90 degrees to account for the fact that the N/S axis is used as reference in stead of E/W axis)

As it is now, if you apply `wind.dirspeed` and then again `wind.uv`  the original `u` an `v` are swapped and have the opposite sign.

Michiel

**Comment by mpvginde** [comment]
@adeckmyn

**Comment by adeckmyn** [comment]
Indeed, that is an error (or at the very least an inconsistency). The function is rarely used (I think), so it has been there for many years. Which of course means that changing this "long time behaviour" may actually cause a problem for a few users...
